CHANGELOG.MD

```diff
@@ -0,0 +1,4 @@
+# `master`
```

Comment:
I suggest CHANGELOG to look more like https://github.com/AckeeCZ/terraform-sql-postgresql/blob/master/CHANGELOG.md ;)

Guess this version should be v1.0.0

Reply:
Updated in 9b067d7334fb8c5d290af011440fbf6aa258de98